BepInEx.IL2CPP/ProxyAssemblyGenerator.cs

```diff
@@ -157,26 +173,13 @@ public void GenerateAssembliesInternal(AppDomainListener listener)
 				UnhollowerBaseLib.LogSupport.TraceHandler += s => listener.DoUnhollowerLog(s, LogLevel.Debug);
 				UnhollowerBaseLib.LogSupport.ErrorHandler += s => listener.DoUnhollowerLog(s, LogLevel.Error);
 
-
-				string unityBaseLibDir = Path.Combine(Preloader.IL2CPPUnhollowedPath, "base");
-
-				if (Directory.Exists(unityBaseLibDir))
-				{
-					listener.DoPreloaderLog("Found base unity libraries", LogLevel.Debug);
```

Comment:
The log message is still useful for technical support as it will tell if the user has base unity libraries.
This should be restored at some place in the code.

Reply:
https://github.com/knah/Il2CppAssemblyUnhollower/blob/f7a2cd63dbc574f78c0e7944b53876365af2a317/AssemblyUnhollower/Program.cs#L222

Reply:
That doesn't appear to use Unhollower's `LogSupport` logging system and instead print it all to stdout, which is not captured by disk logger of BepInEx owing to stdout being redirected to the console instead. 

As such, one option here is to make a PR to knah/Il2CppAssemblyUnhollower and use `LogSupport` like in [here for example](https://github.com/knah/Il2CppAssemblyUnhollower/blob/f7a2cd63dbc574f78c0e7944b53876365af2a317/AssemblyUnhollower/Program.cs#L184) or to manually output this on BepInEx side so that this important info ends up into the disk log as well.